client_plugin/utils/config/config.go

```diff
@@ -156,11 +164,12 @@ func InitConfig(defaultConfigPath string, defaultLogPath string, defaultDriver s
 	}
 	configFile := flag.String("config", defaultConfigPath, "Configuration file path")
 	driverName := flag.String("driver", "", "Volume driver")
+	groupId := flag.String("group", "", "Plugin socket group id")
```

Comment:
I would suggest to add support during installation time as well  @https://github.com/vmware/docker-volume-vsphere/blob/master/plugin_dockerbuild/config.json-template#L57 so that user can supply during installation time and avoid observing following error message

```
docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:0.16
0.16: Pulling from vmware/docker-volume-vsphere
1d54a760dc2a: Download complete 
Digest: sha256:5281d47506740ee927001ea5fc16d8584ed7fc04de0fb25969f0b11eb3243345
Status: Downloaded newer image for vmware/docker-volume-vsphere:0.16
Error response from daemon: dial unix /run/docker/plugins/d2f8c8385b924c1ae13e640a538a0a8a324dfa5c53d567188cebbbbca8f4c097/vsphere.sock: connect: no such file or directory
```

I guess with the proposed code user still sees the above error message when there is `/etc/docker-volume-vsphere.conf` is not present.

Reply:
What group ID should we choose. This is a GOS platform specific parameter and can't be included in the plugin specific env at startup. With managed plugin user can't export variables and have those used by the plugin. Plus there is no way to pass values to the plugin on installation. The only way is to allow the user specify a group ID via the plugin config which is what they can create  before installing the plugin.

Reply:
> Plus there is no way to pass values to the plugin on installation.

Just like as we did for log level, we can supply groupID as an installation parameter and the command would be like `docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:latest VDVS_GROUP_ID=<plugin_socket_gid_value>` that avoids showing installation failure. 

Sample command: `docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:latest VDVS_LOG_LEVEL=debug`

Reference: https://github.com/govint/docker-volume-vsphere/blob/b020c2d357f89ee941e8063bfe70bb90d9a61c1c/client_plugin/utils/config/config.go#L160